packages/ember-glimmer/lib/ember-views/target-action-support.js

```diff
@@ -0,0 +1,217 @@
+import { Mixin } from 'ember-metal/mixin';
```

Comment:
It is indeed, the [mixin in ember views](https://github.com/emberjs/ember.js/blob/master/packages/ember-views/lib/mixins/view_target_action_support.js) actually only provided the TargetActionSupport mixin and a couple of CPs. Most of the actual methods for targetable actions in views was implemented in [ember-htmlbars/lib/component.js](https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/component.js#L281). It made sense to me to actually unify these implementations into its own mixin and not pollute the component.js file for glimmer.